conf/homeserver.yaml

```diff
@@ -304,7 +304,7 @@ listeners:
     tls: false
     type: http
     x_forwarded: true
-    bind_addresses: ['::1', '127.0.0.1']
+    bind_addresses: [127.0.0.1]
```

Comment:
In the case of you remove ipv6 listening you also need to avoid that nginx try a ipv6 connection. So you also need explicitly set proxy pass to `127.0.0.1`.

Reply:
I initially set this to localhost instead of 127.0.0.1, which would not break nginx config.
I need to find back the issue I'm kind of remembering about 127.0.0.1 working and localhost failing. Maybe it was not about listening on localhost but it was about connecting to localhost to another app that was listening on 127.0.0.1. Basically another app that is broken by the same thing you are telling that should not be done here.